README.md

```diff
@@ -1,43 +1,16 @@
 <a name="readme-top"></a>
 
-<!--
-!!! IMPORTANT !!!
-This README is an example of how you could professionally present your codebase. 
-Writing documentation is a crucial part of your work as a professional software developer and cannot be ignored. 
-
-You should modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
-
-For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
--->
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ Required ] Please remove the Microverse logo from your readme file, this logo usually serves as a placeholder to show you how and where you can include a logo in the readme file. You can include a logo on the readme file but please make sure it's not the Microverse logo.

## Code to include a logo
```html
  <img src="link_to_logo.png" alt="logo" width="140"  height="auto" />
```